pom.xml

```diff
@@ -3,9 +3,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>cloud.solarwindscloud</groupId>
+  <groupId>cloud.solarwinds</groupId>
   <artifactId>apm-proto</artifactId>
-  <version>1.0.4-SNAPSHOT</version>
+  <version>0.0.0-SNAPSHOT</version>
```

Comment:
It looks like this should be `1.0.5-*`? How are we going backwards?

Reply:
This version will always be set by github action release.yml. So this field now is a placeholder.